mitodl/open-discussions
# fix ocw course delete

#### Pre-Flight checklist


#### What are the relevant tickets?
None

#### What's this PR do?
Course run slugs for ocw courses always have format `courses/{course url substing}`. The --delete command of backpopulate_ocw_next_data currently only works if you set --course-url-substring to  `courses/{course url substing}` instead of just the course url substring. This fixes the issue.

#### How should this be manually tested?
Pick a ocw course published in your local database and search

Run 
docker compose run web ./manage.py backpopulate_ocw_next_data --course-url-substring {the url substring}  --delete

Where {the url substring} is the final url portion of the course on ocw. You can get it by clicking "Take Course" from the drawer or taking the part of the course's learning resource runs slug value that follows the  `courses/` prefix

Verify that the course is removed from search and set to unpublished
